server/src/main/java/com/google/musicanalysis/cache/AnalysisCache.java

```diff
@@ -100,11 +104,23 @@ public static void saveCache() throws InvalidKeyException, IllegalBlockSizeExcep
     }
   }
 
+  public static void cleanDayOldEntries() {
+    long now = Instant.now().getEpochSecond();
+
+    for (Map.Entry cachePair : cacheMap.entrySet()) { 
+      CacheValue currentCachedData = (CacheValue)cachePair.getValue();
+      long timeOfCacheEntry = currentCachedData.timestamp.getEpochSecond();
+      if (now - timeOfCacheEntry >= ONE_DAY_IN_NANOSECONDS) {
```

Comment:
`getEpochSecond()` returns time in seconds. Why are you comparing it to `ONE_DAY_IN_NANOSECONDS`?

Reply:
Forgot to change one day in nano var..good catch